modules/machineset-yaml-rhv.adoc

```diff
@@ -102,20 +103,22 @@ If you include this parameter, you do not need to specify the hardware parameter
 
 <13> Optional: Specify the size of a VM's memory in MiB.
 
-<14> Optional: Root disk of the node.
+<14> Optional: Specify the size of a Virtual machine's guaranteed memory in MiB. This is only valid if the oVirt cluster setting supports memory overcommitment. For more information, see link:https://access.redhat.com/articles/6454811[Guaranteed memory requirements for OpenShift on Red Hat Virtualization clusters].
```

Comment:
On a second thought I would make it clearer:
Specify the size of a Virtual machine's guaranteed memory in MiB. This is the amount of memory that is guaranteed to not be drained by the ballooning mechanism.

And regarding the link please link the RHV docs explaining guaranteed memory and ballooning.
Link the KCS as a note if they are using an RHV version which is older than 4.4.8